packages/playground/client/src/index.ts

```diff
@@ -47,7 +48,8 @@ export interface StartPlaygroundOptions {
 	disableProgressBar?: boolean;
 	blueprint?: Blueprint;
 	onBlueprintStepCompleted?: OnStepCompleted;
-	onBlueprintStepError?: OnStepError;
+	onBlueprintStepsCompleted?: OnStepsCompleted;
```

Comment:
now that I look at these side by side it feels odd that they are structured differently. do you have ideas for names that might be more symmetric?

```
onBlueprintStepCompleted
onBlueprintStepsCompleted
onBlueprintStepsRunning
```

```
onBlueprintStepsStarting
onBlueprintStepsProgress
onBlueprintStepsFinished
```